airflow/providers/databricks/hooks/databricks.py

```diff
@@ -208,10 +213,10 @@ def _do_api_call(self, endpoint_info, json):
             attempt_num += 1
             sleep(self.retry_delay)
 
-    def _log_request_error(self, attempt_num, error):
+    def _log_request_error(self, attempt_num: Any, error: Any) -> None:
```

Comment:
Again, adding `Any` here is causing an impression that we have higher type coverage, but doesn't really contribute real "value".